doc/guide/install.rst

```diff
@@ -540,11 +554,19 @@ Changing the logging level can be done in multiple ways:
 
 All three approaches can also be combined.
 
-`Mamba <https://mamba.readthedocs.io/en/latest/>`_ Instead of Anaconda
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. _conda-instead-of-mamba:
+
+Conda Instead of Mamba
```

Comment:
```suggestion
Conda as an alternative to Mamba
```

Reply:
"Conda Instead of Mamba" is a bit confusing, since it's not clear what is the recommendation and what is the alternative to said recommendation

Reply:
```suggestion
Conda as Alternative to Mamba
```